src/black/lines.py

```diff
@@ -605,7 +605,16 @@ def _maybe_empty_lines(self, current_line: Line) -> tuple[int, int]:  # noqa: C9
             # Consume the first leaf's extra newlines.
             first_leaf = current_line.leaves[0]
             before = first_leaf.prefix.count("\n")
-            before = min(before, max_allowed)
+            before = (
```

Comment:
This change should be applied only to the preview style; you'll need to add a feature to the enum and use Mode here to decide whether to make the change. Look for other recent changes that changed the formatting style for examples.

Reply:
Thanks!! after taking a look at some of the other PRs for preview styles I've added a new feature to the enum, docs, etc and added another condition here to make sure it's in the Mode. I updated the change log to have this change in the preview section as well. Based on the changes in the other PRs I believe I've updated everything necessary but please let me know if I'm missing anything else!